lib/systems/default.nix

```diff
@@ -213,6 +213,10 @@ rec {
           then selectEmulator pkgs
           else throw "Don't know how to run ${final.config} executables.";
 
+        staticEmulator = pkgs:
+          if builtins.elem final.system pkgs.qemu-user-static.passthru.qemuUserPlatforms then
```

Comment:
Is there any reason to check and return null, rather than just always returning qemu-user-static and letting the meta checks produce an error if its an unsupported platform?

Reply:
This is to check whether the emulated platform is supported instead of the host platform. Could you elaborate on what the change would look like?

Reply:
The change would be to have `emulator` and `staticEmulator` have a consistent interface.  So like `emulator`, `staticEmulator` would throw if unsupported, and there'd be a `staticEmulatorAvailable` to mirror `emulatorAvailable`.  I just want to make sure we don't have two interfaces that work differently for such similar concepts.